src/main.js

```diff
@@ -1,7 +1,14 @@
-import Vue from 'vue'
-import App from './App.vue'
+import Vue from 'vue';
+import App from './App.vue';
+import { Button, Dropdown, Select } from 'element-ui';
+import _ from 'lodash';
```

Comment:
You can remove the import of all the lodash functions, we will use them when needed, and only import the function that we need

-> import capitalize from 'lodash/capitalize'